onboarding/style.css

```diff
@@ -37,6 +38,7 @@ button {
     position: relative;
     top: 0px;
     transition: top .3s;
+    font-weight: bold;
```

Comment:
The bold text on the buttons look a little odd compared to the rest of the page- do you agree or do you like it? If you like it, it can stay.